packages/server/src/api/endpoints/customer.ts

```diff
@@ -14,7 +16,6 @@ export function customerEndpoints(server: FastifyInstance): void {
   server.addSchema({
     $id: 'CustomerUpdateBody',
     type: 'object',
-    required: ['email', 'name', 'addressLine1', 'addressLine2', 'zipCode', 'city', 'country'],
```

Comment:
@anbraten Why have you removed the required flags?

Reply:
@anbraten Never mind. I understood it now as you have this `body.email || customer.email`